app/controllers/api_controller/normalizer.rb

```diff
@@ -19,7 +19,7 @@ def normalize_hash(type, obj, opts = {})
       end
 
       attrs.each do |k|
-        value =  normalize_direct(type, k, obj[k])
+        value =  normalize_direct(type, k, obj.kind_of?(ActiveRecord::Base) ? obj.try(k) : obj[k])
```

Comment:
yes, objects or hashes can go through normalizer so we need to handle both.  Thanks.